Lib/webbrowser.py

```diff
@@ -11,13 +11,41 @@
 
 __all__ = ["Error", "open", "open_new", "open_new_tab", "get", "register"]
 
+
 class Error(Exception):
     pass
 
+
 _lock = threading.RLock()
-_browsers = {}                  # Dictionary of available browser controllers
-_tryorder = None                # Preference order of available browsers
-_os_preferred_browser = None    # The preferred browser
+_browsers = {}  # Dictionary of available browser controllers
```

Comment:
Please, remove unrelated changes: inluding spaces and newlines. It damages the git history for no clear benefit.

Plus, it is hard to review: because I have to get through unrelated changes to find actual ones.

Reply:
i used black for formating the code (PEP08).
i will revert the old code base and put my code again.
i would suggest that someone else runs then black (PEP08)